php/convert-phone-number-to-country-name/index.php

```diff
@@ -0,0 +1,77 @@
+<?
+
+require_once('vendor/autoload.php');
+
+use Appwrite\Client;
+use Appwrite\Services\Locale;
+
+
+/*
+Globaly-scoped cache used by function. Example value:
+[
+    {
+        "code": "+1",
+        "countryCode": "US",
+        "countryName": "United States"
+    }
+]
+*/
+$phonePrefixList = null;
+
+return function($req, $res) use ($phonePrefixList) {
+    // Input validation
+    $phoneNumber = null;
+    try {
+        $payload = \json_decode($req['payload'], true);
+        $phoneNumber = \trim($payload['phoneNumber']);
+    } catch(\Exception $err) {
+        \var_dump($err);
```

Comment:
leftover

Reply:
Actually, this is on purpose. If there is some parsing error, developer should see a log (currently in container) to be able to identify it. "Invalid payload" would not be too useful.

If you still think we should remove var_dump, let me know.

Reply:
Ah okay 👍🏻